packages/langium/src/grammar/internal-grammar-util.ts

```diff
@@ -4,14 +4,17 @@
  * terms of the MIT License, which is available in the project root.
  ******************************************************************************/
 
-import type { URI} from 'vscode-uri';
-import type { LangiumDocuments } from '../workspace/documents';
-import type { AstNode } from '../syntax-tree';
-import * as ast from '../grammar/generated/ast';
-import { Utils } from 'vscode-uri';
-import { getDocument, streamAllContents } from '../utils/ast-util';
-import { TypeResolutionError } from './type-system/type-collector/types';
-import { escapeRegExp } from '../utils/regex-util';
+import type { URI } from 'vscode-uri';
+import type { LangiumDocuments } from '../workspace/documents.js';
+import type { AstNode } from '../syntax-tree.js';
+import * as ast from '../grammar/generated/ast.js';
+import vscodeUri from 'vscode-uri';
```

Comment:
This line is throwing errors when trying to bundle:

```ts
import vscodeUri from 'vscode-uri';
```

```
"default" is not exported by "node_modules/.pnpm/vscode-uri@3.0.7/node_modules/vscode-uri/lib/esm/index.js", imported by "node_modules/.pnpm/langium@2.0.0-next.239179f/node_modules/langium/lib/grammar/internal-grammar-util.js".
file: node_modules/.pnpm/langium@2.0.0-next.239179f/node_modules/langium/lib/grammar/internal-grammar-util.js:7:7
5:  ******************************************************************************/
6: import * as ast from '../grammar/generated/ast.js';
7: import vscodeUri from 'vscode-uri';
          ^
8: import { getDocument, streamAllContents } from '../utils/ast-util.js';
9: import { TypeResolutionError } from './type-system/type-collector/types.js';
```

Reply:
Well, that's just great... Without using the default import, it doesn't bundle in esbuild - but with it, it doesn't seem to bundle in vite...

Reply:
Rollup to be more specific :)